packages/bitcoin/redux/actions.js

```diff
@@ -40,6 +40,14 @@ export const bitcoinTxDraftUpdateFee = ({ address, parentAddress, fee }) => ({
   parentAddress,
 })
 
+export const bitcoinTxCreateHistory = ({ address, parentAddress, latestTxDate, txList }) => ({
+  type: ActionsTypes.BITCOIN_TX_GET_HISTORY,
```

Comment:
Action name is 'bitcoinTxCreateHistory', but constant name is 'BITCOIN_TX_GET_HISTORY'
May leads to inconvenience.